pkg/status/status.go

```diff
@@ -26,8 +27,21 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/version"
 )
 
+var startTime = time.Now()
 var timeFormat = "2006-01-02 15:04:05.000000 MST"
 
+// FormatUptime formats a duration into a human-readable, uptime-like string
```

Comment:
As the start time will be used to diagnose problems, I'm wondering if having a more raw display would be more efficient (ie. 2006-01-02 15:04:05.000000 MST), what do you think?

Reply:
Removed the custom formatting